tools/agent-prebuilt.sh

```diff
@@ -84,35 +88,51 @@ function do_clone {
             echo "Please commit these before attempting to rebase."
             return 1
         fi
-        git pull --rebase
+
+        git fetch
```

Comment:
Let's make that `git fetch -p origin` perhaps?  `-p` is to prune branches from the origin that are gone now (e.g. old PR and feature branches).  I think hardcoding "origin" as the remote will always be correct for us.

Reply:
That sounds reasonable.